### DISCUSSION TITLE
I made a Ruby API client for Umami

### DISCUSSION THREAD

**Discussion by rameerez** [umami-software/umami#2852] [Show and tell]
### Describe the feature or enhancement

Hi! Love Umami, thanks for making it!

I've made a Ruby wrapper for the Umami API that supports both Umami Cloud and self-hosted instances. I hope it's useful for all the Rails folks out there using Umami for web analytics.

I've just built it so testing and PRs with improvements are welcome! https://github.com/rameerez/umami-ruby

**Comment by mikecao** [comment]
Thanks! You can submit a PR for the website and we'll get it added here, https://umami.is/docs/integrations

**Reply by rameerez** [reply]
I opened a PR, thanks! https://github.com/umami-software/website/pull/182
